### PR TITLE
Added Table support

### PR DESCRIPTION
**Pull request by mirandarfsm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,15 @@ npm install
...
 
+## Run example
+
+```
+python3 -m http.server
+
+```
+
+[StackOverflow: Start a server](https://stackoverflow.com/a/21608670)
+
```
Comment (koddsson) [inline README.md:65]: I think starting a local server is up to the developer. I use the `http-server` npm package myself.

```suggestion
```

File: src/index.ts [hunk 2]
```diff
@@ -299,6 +302,19 @@ if (!window.customElements.get('md-strikethrough')) {
   window.customElements.define('md-strikethrough', MarkdownStrikethroughButtonElement)
 }
 
+class MarkdownTableButtonElement extends MarkdownButtonElement {
+  constructor() {
+    super()
+    const tableMarkdown = '| header | header |\n| ------ | ------ |\n| cell | cell |\n| cell | cell |\n'
```
Comment (koddsson) [inline src/index.ts:308]: What if a user wants to create a table with three or more columns?

**Review by koddsson — COMMENTED** [review]
Thank you for the pull request @mirandarfsm! I think it tries to solve a real problem for GitHub Markdown users that GitHub doesn't have a great UI for right now.

With that being said, I'm not sure this is the solution we want to go for right now. Making a table is more involved than bolding text or creating headers, and I think we'd like to tackle the problem differently for GitHub.

**Comment by mirandarfsm** [comment]
Hello @koddsson, i apologize for the delay in replying, i was not notify by github.
I got it. I needed this change to my own project. I made the change and push it to you. I think this could help. 
I let this change here, if you'd like...
Thank you for the anwser.

File: src/index.ts (hunk 2 above)
Comment (mirandarfsm) [reply src/index.ts:284]: This part add a Minimum to start a table. You can add more column like github's markdown do that.

Example:

3 columns

'| header | header | header |
| ------ | ------ | ------ |
| cell | cell | cell |
| cell | cell | cell |'

| header | header | header |
| ------ | ------ | ------ |
| cell | cell | cell |
| cell | cell | cell |

3 lines

'| header | 
| ------ | 
| cell | 
| cell | 
| cell | '

| header | 
| ------ | 
| cell | 
| cell | 
| cell |

File: README.md (hunk 1 above)
Comment (mirandarfsm) [reply README.md:65]: You can ignore this part. I put it to help me.

**Comment by camertron** [comment]
Closing per @koddsson's comment.

**Pull request closed without merging**